src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/TestHelpers.cs

```diff
@@ -56,4 +59,30 @@ internal static class TestHelper
                 null
                 );
     }
+
+    //This is to make sure normalising the keys done correctly with culture invariant
+    //Lower case of SETTINGSKEY in Turkish is not settingskey but settıngskey
+    //https://en.wikipedia.org/wiki/Dotted_and_dotless_I 
+    public class TemporaryCultureSwitch : IDisposable
```

Comment:
This a general-purpose test class, so it might be better to put it in the shared `TestInfrastructure` project - we're more likely to remember it when writing other tests.